miflora/miflora_poller.py

```diff
@@ -104,6 +104,7 @@ def fill_cache(self):
                         datetime.now() - self._cache_timeout + timedelta(seconds=300)
                     )
                     return
+            time.sleep(1) # without this, the previous write_handle is done after we read the data
```

Comment:
That's a poor fix, I know that. But I don't really know much about Python or the Bluethooth libaray. It should be checked whether the handle has already been written.